pcweb/pages/customers/stories/bayesline.md

```diff
@@ -0,0 +1,106 @@
+---
+company: Bayesline
+description: "How Bayesline builds 4x faster using Reflex instead of Plotly Dash"
+domain: "https://bayesline.com/"
+founded: "New York, 2024"
+investors: "Y Combinator",
+meta: [
+  {
+    "name": "keywords",
+    "content": "
+      plotly dash,
+      python and react,
+      python and react js,
+      python react,
+      python web app,
+      python web app framework,
+      python web apps,
+      react js python,
+      react js with python,
+      react python,
+      react python websocket,
+      react with python,
+      web app framework python,
+      web app python,
+    "
+  }
+]
+---
+
+Why Bayesline chose Reflex instead of Plotly Dash
+
+## TLDR;
+
+Sebastian and his small team use Reflex to keep the full flexibility of a traditional ReactJS app **WITHOUT** needing to:
+
+* Learn frontend: React (JavaScript), NodeJS, TailwindCSS
+
+```md quote Misha
+Reflex definitely saved us from needing to hire a frontend engineer.
+```
+
+* Write boilerplate to stitch together their frontend and backend:
+
+```md quote Sebastian
+50% less code than Dash
+```
+
+* Maintain expensive, fragile, and inevitably slow Dash apps; using Reflex instead of Plotly Dash
+
```

Comment:
This should be "Sped us up by 4x relative to learning React", the lego quote is great, but not a tangible take away of what was gained I think

Reply:
and here I would mention "Production Grade Web App Software in Pure Python" as it is a great comparison to React that can also produce production grade web apps